backend/open_webui/retrieval/utils.py

```
@@ -247,51 +309,116 @@ def get_embedding_function(
     embedding_batch_size,
 ):
     if embedding_engine == "":
-        return lambda query: embedding_function.encode(query).tolist()
+        return lambda query, user=None: embedding_function.encode(query).tolist()
```

Ruff (0.12.2)

Unused lambda argument: `user`

(ARG005)